helpdesk_stock/views/helpdesk_views.xml

```diff
@@ -0,0 +1,136 @@
+                                name="action_inventory_cancel" string="Cancel"
+                                type="object"
+                                attrs="{'invisible': [('inventory_stage', '!=', 'requested')]}"/>
+                        <button id="action_reset_draft"
+                                name="action_inventory_reset" string="Reset to Draft"
+                                type="object"
+                                attrs="{'invisible': [('inventory_stage', '!=', 'cancelled')]}"/>
+                        <field name='inventory_stage' widget="statusbar"/>
+                    </header>
+                    <field name="team_id" invisible="1"/>
+                    <group groups="stock.group_stock_user">
+                        <group id="stock-left">
+                            <field name="warehouse_id"
+                                   groups="stock.group_stock_multi_warehouses"/>
+                            <field name="inventory_location_id"
+                                   domain="[('usage', 'in', ['internal', 'supplier', 'customer'])]"/>
+                        </group>
+                        <group id="stock-right"></group>
+                    </group>
+                    <field name="line_ids" nolabel="1">
+                        <tree editable="bottom">
+                            <field name="sequence" widget="handle"/>
+                            <field name="product_id"
+                                   context="{'quantity': qty_ordered,
+                                             'uom': product_uom_id}"/>
+                            <field name="name"/>
+                            <field name="route_id" />
+                            <field name="qty_requested"
+                                   string="Requested Qty"
+                                   context="{'quantity': qty_requested,
+                                             'uom': product_uom_id}"/>
+                            <field name="qty_ordered"
+                                   string="Ordered Qty"
+                                   context="{'quantity': qty_ordered,
+                                             'uom': product_uom_id}"
+                                   attrs="{'invisible': [('state', '=', 'draft')]}"/>
+                            <field name="qty_delivered"
+                                   attrs="{'invisible': [('state', '=', 'draft')]}"/>
+                            <field name="product_uom_id"
+                                   attrs="{'readonly': [('state', 'in', ('sale', 'done', 'cancel'))]}"
+                                   options="{'no_open': True, 'no_create': True}"
+                                   groups="uom.group_uom"/>
+                            <field name="state" readonly="1"/>
+                        </tree>
+                    </field>
+                </page> -->
+                <page string="Stock Requests">
```

Comment:
@murtuzasaleh Please delete the commented lines and rename this page "Inventory". Thank you.